### ISSUE TITLE
Windows Java Test failure

### ISSUE THREAD

**Issue by jakelandis** [elastic/logstash#7799]
It seems that the benchmark-cli tests are having issues on Windows.  Note, the Java side of tests worked on Windows prior. 

cc: @original-brownbear 
```
org.logstash.benchmark.cli.LsMetricsMonitorTest > parsesCpuUsage FAILED

    java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/Users/jenkins/workspace/elastic+logstash+pull-request+multijob-windows-compatibility/tools/benchmark-cli/build/resources/test/org/logstash/benchmark/cli/metrics.json
        at sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
        at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
        at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
        at sun.nio.fs.WindowsPath.parse(WindowsPath.java:94)
        at sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:255)
        at java.nio.file.Paths.get(Paths.java:84)
        at org.logstash.benchmark.cli.LsMetricsMonitorTest.parsesCpuUsage(LsMetricsMonitorTest.java:59)

org.logstash.benchmark.cli.LsMetricsMonitorTest > parsesFilteredCount FAILED

    java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/Users/jenkins/workspace/elastic+logstash+pull-request+multijob-windows-compatibility/tools/benchmark-cli/build/resources/test/org/logstash/benchmark/cli/metrics.json
        at sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
        at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
        at sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
        at sun.nio.fs.WindowsPath.parse(WindowsPath.java:94)
        at sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:255)
        at java.nio.file.Paths.get(Paths.java:84)
        at org.logstash.benchmark.cli.LsMetricsMonitorTest.parsesFilteredCount(LsMetricsMonitorTest.java:35)
```

**Comment by original-brownbear** [comment]
@jakelandis argh yea there's this and even one additional Windows incompatibility in the benchmark code.
Will fix before you wake up :)

**Comment by original-brownbear** [comment]
Fixed in #7800 :)
